core/internal/runbranch/fork_test.go

```diff
@@ -0,0 +1,51 @@
+package runbranch_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/wandb/wandb/core/internal/gqlmock"
+	"github.com/wandb/wandb/core/internal/runbranch"
+)
+
+func TestForkSameRunIDs(t *testing.T) {
+	ctx := context.TODO()
```

Comment:
tests should use `context.Background()`, not `TODO()` (see the docs on the context package)